package.json

```diff
@@ -1,12 +1,11 @@
 {
   "devDependencies": {
-    "@symfony/webpack-encore": "^0.28.0",
-    "bootstrap": "^4.1.3",
-    "date-fns": "^2.0.0",
-    "jquery": "^3.3.1",
-    "node-sass": "^4.9.4",
+    "@symfony/webpack-encore": "^1.0.5",
+    "bootstrap": "^4.6.0",
+    "date-fns": "^2.17.0",
+    "node-sass": "^5.0.0",
```

Comment:
with node-sass being deprecated, I think migrating to the `sass` package would be better actually